libs/server/Resp/BasicCommands.cs

```diff
@@ -366,24 +332,18 @@ private bool NetworkSETEX<TGarnetApi>(bool highPrecision, ref TGarnetApi storage
             }
 
             var val = parseState.GetArgSliceByRef(2).SpanByte;
-            var valPtr = val.ToPointer() - (sizeof(int) + sizeof(long));
-            var vSize = val.Length;
-
-            // Save prior state on network buffer
-            var save1 = *(int*)valPtr;
-            var save2 = *(long*)(valPtr + sizeof(int));
 
-            *(int*)valPtr = vSize + sizeof(long); // expiry info
-            SpanByte.Reinterpret(valPtr).ExtraMetadata = DateTimeOffset.UtcNow.Ticks +
-                                                         (highPrecision
-                                                             ? TimeSpan.FromMilliseconds(expiry).Ticks
-                                                             : TimeSpan.FromSeconds(expiry).Ticks);
+            var newValLen = val.Length + sizeof(long);
```

Comment:
Since the entire command has been received, there is no need to "save" (save1 and save2) as your PR correctly does. However, we cannot stack-allocate values like this. We would like to use the existing buffer so that we avoid data movement. For cleaner code, we might want to make this a helper method (shift and add metadata).